assets/src/js/admin/onboarding-wizard/components/step-link/index.js

```diff
@@ -0,0 +1,60 @@
+// Import vendor dependencies
+import PropTypes from 'prop-types';
+
+// Import store dependencies
+import { useStoreValue } from '../../app/store';
+
+// Import utilities
+import { goToStep } from '../../app/store/actions';
+
+// Import styles
+import './style.scss';
+
+const StepLink = ( { title, step } ) => {
+	const [ { currentStep }, dispatch ] = useStoreValue();
+
+	const check = <svg width="27" height="22" viewBox="0 0 27 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Similar to the chevron, I think the relative complexity of an SVG deserves its own component.

Reply:
Agree with this thinking, I've gone ahead and moved the Checkmark to its own component. Ref: https://github.com/impress-org/givewp/pull/4942/commits/f0711317f5af138dc87f4a287f62c127af7efa0f